datahub-web-react/src/app/entity/dataset/profile/schema/utils/types.ts

```diff
@@ -8,3 +8,8 @@ export interface ExtendedSchemaFields extends SchemaField {
     isNewRow?: boolean;
     isDeletedRow?: boolean;
 }
+
+export enum SchemaBlameViewType {
```

Comment:
Would this be better named "SchemaViewType"?

Reply:
Agreed